backend/backend.py

```diff
@@ -3,13 +3,13 @@
 import bcrypt
 from bson.objectid import ObjectId
 from flask import Flask, request, flash, jsonify, redirect, url_for
-from flask_login import (
+'''from flask_login import (
     LoginManager,
     current_user,
     login_required,
     login_user,
     logout_user
-)
+)'''
```

Comment:
I'm currently working on implementing user-authentication with Flask-Login so we will need this.